packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -51,9 +53,22 @@ const start = async (zcf, privateArgs) => {
   /**
    * Used to look up the brands for keywords, excluding RUN because it's a special case.
    *
-   * @type {MapStore<Keyword, [brand: Brand, liquidityBrand: Brand]>}
+   * @type {MapStore<Keyword, Brand>}
    */
   const brandForKeyword = makeStore('brands');
+  /**
+   * @type {MapStore<Brand, Brand>}
+   */
+  const liquidityBrandForBrand = makeStore('liquidityBrands');
```

Comment:
The argument to `makeStore()` is supposed to describe the **key**, not the **value**.